src/Tribe/Admin/Notice/Install_Event_Tickets.php

```diff
@@ -192,7 +193,67 @@ public function should_display_notice_activate(): bool {
 		 *
 		 * @param bool $should_display True if the notice should display.
 		 */
-		return apply_filters( 'tec_events_admin_notice_event_tickets_should_display', $this->is_installed() && ! $this->is_active() && ! $this->is_install_plugin_page() );
+		return apply_filters( 
```

Comment:
For readability, we should put these conditionals into a variable, and then pass that variable to the `apply_filters()` call. With this many conditions strung together, it can be hard to mentally parse.